libfastjson.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: libfastjson
+  version: 1.2304.0
+  epoch: 0
+  description: a fast json library for C
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3a8402c1de7c7747c95229db26d8d32fb85a7a52
+      repository: https://github.com/rsyslog/libfastjson
+      tag: v${{package.version}}
+
+  - runs: |
+      libtoolize --copy
+      aclocal -I m4 --install
+      autoheader
+      autoconf
+      automake --foreign --add-missing --force-missing --copy
+      ./configure --prefix=/usr
+      make -j $(nproc)
```

Comment:
We can use our `autoconf/make` pipeline here

Reply:
I sure wish our `autoconf/make` pipeline did `-j $(nproc)` by default when building locally for larger builds...  (Not this one, but others...)

Reply:
It does :) - https://github.com/chainguard-dev/melange/blob/4ecc73b9b01f3a4f996836ccbc10dcac60ea3014/pkg/build/pipelines/autoconf/make.yaml#L20

Reply:
Okay, updated to use `autoconf/make`